readme.md

```diff
@@ -177,6 +179,31 @@ the anecdote, you can throw away the data."
 [R: verywell.com](https://www.verywell.com/what-is-the-hawthorne-effect-2795234)
 
 
+#### [Munchausen syndrome](https://en.wikipedia.org/wiki/Factitious_disorder_imposed_on_self)
+
+> Person repeatedly and deliberately acts as if he or she has a physical or mental illness when he or she is not really sick.
+
+![Image](./img/Munchausen_syndrome.png)
+
+Munchausen's syndrome is a psychological and behavioural condition where someone pretends to be ill or induces symptoms of illness in themselves. It is also sometimes known as factitious disorder. People with the condition intentionally produce or pretend to have physical or psychological symptoms of illness. Their main intention is to assume the "sick role" to have people care for them and be the centre of attention. 
+
+[R: nhs.uk](https://www.nhs.uk/conditions/munchausens-syndrome/)
+
+[R: my.clevelandclinic.org](https://my.clevelandclinic.org/health/articles/munchausen-syndrome)
+
+
+#### [Munchausen syndrome by proxy](https://en.wikipedia.org/wiki/Factitious_disorder_imposed_on_another)
+
+> Person acts as if an individual he or she is caring for has a physical or mental illness when the person is not really sick.
+
+![Image](./img/Munchausen_syndrome-by-proxy.png)
+
+Munchausen by proxy syndrome is a relatively rare form of child abuse that involves the exaggeration or fabrication of illnesses or symptoms by a primary caretaker. The adult perpetrator lies about illness in another person under his or her care, usually a child under 6 years of age. "By proxy" indicates that a parent or other adult is making up or exaggerating symptoms in a child, not in himself or herself.
+
+[R: .healthline.com](https://www.healthline.com/health/munchausen-syndrome-by-proxy)
+
+[R: .medicinet.com](https://www.medicinenet.com/script/main/art.asp?articlekey=38327)
```

Comment:
This ref doesn't cut it. Please find a better ref. :slightly_smiling_face:

Reply:
Okay. :+1: